deno/deepgram-topic-detection/README.md

```diff
@@ -0,0 +1,47 @@
+```
+
+## 📝 Environment Variables
+
+A single environment variable: DEEPGRAM_API_KEY
```

Comment:
Please remove references to "Environment" since environment variables are for older versions of open runtimes:

```suggestion
## 📝 Variables

A single variable: `DEEPGRAM_API_KEY`
```